recipes/grayskull/meta.yaml

```diff
@@ -0,0 +1,57 @@
+{% set name = "grayskull" %}
+{% set version = "0.1.1" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 654485aae458db4565eee55beaae565bd1417dee266faace458f4fce0f750ad2
+
+build:
+  number: 0
+  noarch: python
+  entry_points:
+    - grayskull = grayskull.__main__:main
+    - gs = grayskull.__main__:main
```

Comment:
As a ghostscript user I'm not really ok with the `gs` short version. However, I understand that even the long `grayskull` command is a temporary name, right?

Reply:
Yes, but I can remove it if you want to